docs/source/users/BenchmarkProblems.rst

```diff
@@ -146,10 +187,41 @@ The WISH problems find 5 unknown parameters, and fit to 512 data points.
    |MantidWarning|
 
 
+NIST
+====
+
+**Download** :download:`.zip <http://numerical.rl.ac.uk/fitbenchmarking/NIST.zip>`
+or :download:`.tar.gz <http://numerical.rl.ac.uk/fitbenchmarking/NIST.tar.gz>`
+
+These problems (also found in the folder `examples/benchmark_problems/NIST`) contain
+data from the `NIST Nonlinear Regression <https://www.itl.nist.gov/div898/strd/nls/nls_main.shtml>`_ test set.
+
+These problems are split into low, average and high difficulty.
+They have between 2 and 9 unknown parameters, and
+fit between 6 and 250 data points.
+
+
+Poisson Data
+============
+
+**Download** :download:`.zip <http://numerical.rl.ac.uk/fitbenchmarking/Poisson.zip>`
+or :download:`.tar.gz <http://numerical.rl.ac.uk/fitbenchmarking/Poisson.tar.gz>`
+
+These problems (also found in the folder `examples/benchmark_problems/Poisson`) contain
+both simulated and real data measuring particle counts. The real data is ISIS
+muon data, and the simulated datasets have been made to represent counts using
+models provided by both Mantid and Bumps.
+
+These problems have between 4 and 6 unknown parameters, and around 350, 800,
+and 2000 data points for simulated bumps, HIFI_160973, and simulated mantid
+respectively.
+
```

Comment:
```suggestion
.. warning::   
   |MantidWarning|
```

Reply:
Sorry, I think I got the whitespace wrong here -- it comes out like: 
![image](https://user-images.githubusercontent.com/13645545/151143965-31d2552a-6559-43fc-9272-881b6b9e19da.png)